Dockerfile.nginx.config

```
@@ -0,0 +1,19 @@
+# used for nginx container
+
+server {
+    listen 80;
```

I try to avoid Priviliged Ports, so I would switch this to 8080.
```suggestion
    listen 8080;
```